documentation/modules/con-standard-infra-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-plans.adoc
+
+[id='con-standard-infra-config-{context}']
+= Standard infrastructure configuration
+
+`StandardInfraConfig` resources is used to configure infrastructure deployed by `standard` address
+spaces. The standard infra configuration is referenced by address space plans using a `defined-by`
+annotation. For more information, see xref:con-address-space-plans-{context}[].
+
+[source,yaml,options="nowrap"]
+.link:resources/standard-infra-config-example.yaml[standard-infra-config-example.yaml]
+----
+include::../common/standard-infra-config-example.yaml[]
+----
+
+The `version` field represents the {ProductName} version used. When upgrading, {ProductName} will
```

Comment:
Change "represents" to "specifies" (also in the following sentences)

When upgrading, {ProductName} uses this field to determine whether to upgrade the infrastructure to the requested version. (?Did I capture the meaning correctly?)